### PR TITLE
dist: provide sha256/sha512 hashes

### PR DESCRIPTION
**Pull request by pquentin**
### Description

Follow new Apache release policy: https://www.apache.org/dev/release-distribution.html#sigs-and-sums

### Status

- done, ready for review

### Checklist (tick everything that applies)

- [x] [Code linting](http://libcloud.readthedocs.org/en/latest/development.html#code-style-guide) (required, can be done after the PR checks)
- [x] Documentation
- [x] [Tests](http://libcloud.readthedocs.org/en/latest/testing.html)
- [x] [ICLA](http://libcloud.readthedocs.org/en/latest/development.html#contributing-bigger-changes) (required for bigger changes)

### CODE REVIEW

**Comment by Kami** [comment]
Haven't tested it, but LGTM :+1:

File: dist/hash.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+
```
Comment (Kami) [inline dist/hash.py:3]: Wouldn't hurt to add a docstring which describes what this script does.

Comment (pquentin) [reply dist/hash.py:3]: Thanks, done.

**Comment by pquentin** [comment]
It's quite easy to test with `./release.sh -u $user@apache.org` from dist, assuming you have a GPG key associated with `$user@apache.org`.

**Pull request merged**